subsys/net/ip/Kconfig

```diff
@@ -437,17 +437,40 @@ config NET_BUF_TX_COUNT
 	  Each data buffer will occupy CONFIG_NET_BUF_DATA_SIZE + smallish
 	  header (sizeof(struct net_buf)) amount of data.
 
+choice
+	prompt "Network packet data allocator"
```

Comment:
Apparently should be "Network packet data allocator *type*".

Reply:
sure